pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -44,6 +44,15 @@ def capsule_host(capsule_factory):
     Broker(hosts=[new_cap]).checkin()
 
 
+@pytest.fixture
+def large_capsule_host(capsule_factory):
+    """A fixture that provides a Capsule based on config settings"""
+    new_cap = capsule_factory(deploy_flavor=settings.flavors.custom_db)
```

Comment:
Large flavor should have been used here instead of custom_db, or is this intended?
```suggestion
    new_cap = capsule_factory(deploy_flavor=settings.flavors.large)
```

Reply:
@vsedmik - specifically for DISK-only. There is a capability to deploy any VM and then increase the disk only with add-disk. I can't recall, off the top of my head, if that flavor was made specifically for the scope of this test.

If it was not and that isn't a satqe-ssd flavor, add-disk "methodoloy" should be used here.

Reply:
Well, it's not DISK-only, the concurrent sync consumes other resources too.